external-reports/src/test/java/org/sonar/java/externalreport/SpotBugsSensorTest.java

```diff
@@ -108,7 +108,7 @@ public void issues_with_sonarqube_72() throws IOException {
     ExternalIssue first = externalIssues.get(0);
     assertThat(first.primaryLocation().inputComponent().key()).isEqualTo("spotbugs-project:src/main/java/org/myapp/Main.java");
     assertThat(first.ruleKey().rule()).isEqualTo("HE_EQUALS_USE_HASHCODE");
-    assertThat(first.ruleKey().repository()).isEqualTo("spotbugs");
+    assertThat(first.ruleKey().repository()).isEqualTo("external_spotbugs");
```

Comment:
FYI: this change is ok because of a change on SensorContextTester side : so only the way to test things has changed with upgrade of API.